e2e/cypress/integration/api/iam/10_projects_api_action_update.ts

```diff
@@ -0,0 +1,157 @@
+import { describeIfIAMV2p1 } from '../../constants';
```

Comment:
just noticed, these should be `.spec.ts` files, not `.ts`

Reply:
Thank you! I could not figure out why they looked different in VS code. 
<img width="304" alt="Screen Shot 2019-11-20 at 4 44 28 PM" src="https://user-images.githubusercontent.com/1679247/69290514-10610000-0bb5-11ea-983e-c242514a02d9.png">

Reply:
I renamed them all in the three PRs I have out.